### ISSUE TITLE
[Bug]: pimcore-install doesn't use own --env parameter

### ISSUE THREAD

**Issue by dkarlovi** [pimcore/pimcore#11912]
### Expected behavior

```
vendor/bin/pimcore-install --env=test # test env
```

### Actual behavior

```
vendor/bin/pimcore-install --env=test # uses whichever env is set in APP_ENV environment variable
```

### Steps to reproduce

Described above.

Note that the param is documented in help
```
$ vendor/bin/pimcore-install help

(...) 

  -e, --env=ENV         The Environment name. [default: "dev"]

```

**Comment by dkarlovi** [comment]
Somewhat related to #11247 maybe.

**Comment by matthiashamacher** [comment]
Even though it is documented in the help it is intended that it does not work. At least as I understand the comment in the installer: https://github.com/pimcore/pimcore/issues/10317#issuecomment-938034045

**Comment by dkarlovi** [comment]
In that case, the command shouldn't expose the parameter, it should remove it.

Currently it accepts the parameter but ignores it, meaning you can easily trash your `dev` database because you ran it with `--env test` instead of `APP_ENV=test`.

**Comment by matthiashamacher** [comment]
I agree, but I don't think there is a possibility to remove that as it comes from Symfony and is by default in all commands...

**Comment by dkarlovi** [comment]
IMO since [the kernel environment is configured from the command directly](https://github.com/pimcore/pimcore/blob/d124b16a837567ed9b7ff010fdeff6e82e9bfaf3/bin/pimcore-install#L58), I see no reason why the environment wouldn't be read correctly.

I don't see any reason why these would result in any sort of a difference:

```
vendor/bin/pimcore-install --env=test
APP_ENV=test vendor/bin/pimcore-console
```

If changing the environment is not supported (again, I don't see why it wouldn't be), then neither the environment variable should be read. If changing the environment is supported, then it shouldn't matter how the environment to use was specified, env var or parameter.

**Comment by brusch** [comment]
@dkarlovi @matthiashamacher The following code 
https://github.com/pimcore/demo/blob/c7ba7363f5da083bcdc80c35e947f7da602e2936/bin/console#L36-L43
needs to be considered here: 
https://github.com/pimcore/pimcore/blob/54ff7a47658ddbbaa66281cd48957e378ae54cd9/bin/pimcore-install#L58

Then this should work - can someone put that into a PR? 🤗 Thanks in advance!

**Comment by brusch** [comment]
Fixed by #11921
